linera-views/src/backends/scylla_db.rs

```diff
@@ -822,35 +891,51 @@ impl AdminKeyValueStore for ScyllaDbStoreInternal {
         namespace: &str,
     ) -> Result<(), ScyllaDbStoreInternalError> {
         Self::check_namespace(namespace)?;
-        let session = SessionBuilder::new()
-            .known_node(config.uri.as_str())
-            .build()
-            .boxed()
-            .await?;
-        // Create a keyspace if it doesn't exist
-        let query = format!(
-            "CREATE KEYSPACE IF NOT EXISTS kv WITH REPLICATION = {{ \
-                'class' : 'NetworkTopologyStrategy', \
-                'replication_factor' : {} \
-            }}",
-            config.common_config.replication_factor
-        );
-
-        // Execute the query
-        let prepared = session.prepare(query).await?;
-        session.execute_unpaged(&prepared, &[]).await?;
+        let session = build_session(&config.uri).await?;
 
-        // Create a table if it doesn't exist
-        // The schema appears too complicated for non-trivial reasons.
-        // See TODO(#1069).
-        let query = format!(
-            "CREATE TABLE kv.{} (root_key blob, k blob, v blob, primary key (root_key, k))",
-            namespace
-        );
+        // Create a keyspace if it doesn't exist
+        let statement = session
+            .prepare(format!(
+                "CREATE KEYSPACE IF NOT EXISTS kv WITH REPLICATION = {{ \
+                    'class' : 'NetworkTopologyStrategy', \
+                    'replication_factor' : {} \
+                }}",
+                config.common_config.replication_factor
+            ))
+            .await?;
+        session
+            .execute_single_page(&statement, &[], PagingState::start())
+            .await?;
 
-        // Execute the query
-        let prepared = session.prepare(&*query).await?;
-        session.execute_unpaged(&prepared, &[]).await?;
+        let statement = session
+            .prepare(format!(
+                "CREATE TABLE kv.{} (\
+                    root_key blob, \
+                    k blob, \
+                    v blob, \
+                    PRIMARY KEY (root_key, k) \
+                ) \
+                WITH compaction = {{ \
+                    'class'            : 'SizeTieredCompactionStrategy', \
+                    'min_sstable_size' : 52428800, \
+                    'bucket_low'       : 0.8, \
+                    'bucket_high'      : 1.25, \
+                    'min_threshold'    : 4, \
+                    'max_threshold'    : 8 \
+                }} \
+                AND compression = {{ \
+                    'sstable_compression': 'LZ4Compressor', \
+                    'chunk_length_in_kb':'4' \
+                }} \
+                AND caching = {{ \
+                    'enabled': 'true' \
+                }}",
```

Comment:
Some documentation of this would be nice as well. Why are those parameters chosen that way?